plenum/server/replicas.py

```diff
@@ -73,15 +76,17 @@ def _master_replica(self):
 
     def service_inboxes(self, limit: int = None):
         number_of_processed_messages = \
-            sum(replica.serviceQueues(limit) for replica in self._replicas)
+            sum(replica.serviceQueues(limit) for replica in self._replicas.values())
         return number_of_processed_messages
 
     def pass_message(self, message, instance_id=None):
```

Comment:
Also `Node.msgHasAcceptableInstId` should be corrected. It must stash messages for instances with a number higher than the current maximum required number.

Reply:
Done